VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -90,19 +99,19 @@ final internal class ResponseCache
             self.cache.setObject(responseDictionary, forKey: key)
         }
         
-        private func responseDictionaryForKey(key: String) -> VimeoClient.ResponseDictionary?
+        private func responseDictionary(forKey key: String, completion: ResponseDictionaryClosure)
```

Comment:
Why not use a return value instead of a closure? Return value is logically simpler and there doesn't seem to be an async aspect to this.